src/Kunstmaan/GeneratorBundle/Resources/SensioGeneratorBundle/skeleton/searchpage/Resources/views/Pages/SearchPage/view.html.twig

```diff
@@ -40,8 +40,7 @@
                 {% if q_query is defined and q_query != '' %}
                     <p class="search-results__text">
                         {% set nrResults = pagerfanta.getNbResults() %}
-                        <strong>{{ '{0} search.no_results|{1} one.result|]1,Inf[ many.results.%count%.%search_for%'
-                            |transchoice(nrResults, {'%count%': nrResults, '%search_for%': q_query}) }}</strong>
+                        <strong>{{ nrResults }} {% if nrResults == 1 %}{{ 'search.result'|trans }}{% else %}{{ 'search.results'|trans }}{% endif %}</strong> {{ 'search.for'|trans }} <strong>{{ q_query }}</strong>
```

Comment:
I saw in the old transchoice that there was an option for if there were no found results. Is this option unneccesary because your fix does not include this translation anymore.

Reply:
I did it this way to keep the translation text "simple" as our translation interface and export/import makes the edit of the new syntax quite hard to edit.

This split in translations would show "1 result" / "0 results" / "15 results"